### PR TITLE
ADD: Enable react-native-community eslint plugin

### PR DESCRIPTION
**Pull request by limpbrains**
followup #5623 
- enable @react-native-community eslint plugin
- fix all related errors
- upgrade eslint, plugins, typescript

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/rgwobyzict46qg2pndl5qslr6y?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/11453)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] 

[download apk](https://lambda-download-android-build.herokuapp.com/download/11456)

**Comment by Overtorment** [comment]
90 files changed o_O

**Comment by limpbrains** [comment]
> 90 files changed o_O

Психанул

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/avkh752oozphfthy4yo7rk6oha?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/11469)

**Review by marcosrdz — APPROVED** [review]

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

File: UnlockWith.js [hunk 1]
```diff
@@ -44,17 +44,16 @@ const UnlockWith = () => {
   const colorScheme = useColorScheme();
 
   const initialRender = async () => {
-    let biometricType = false;
+    let bt = false;
```
Comment (Overtorment) [inline UnlockWith.js:47]: curious, why this change?

File: blue_modules/currency.js [hunk 2]
```diff
@@ -97,7 +97,7 @@ async function updateExchangeRate() {
   } catch (Err) {
     console.log('Error encountered when attempting to update exchange rate...');
     console.warn(Err.message);
-    const rate = JSON.parse(await AsyncStorage.getItem(EXCHANGE_RATES_STORAGE_KEY));
+    rate = JSON.parse(await AsyncStorage.getItem(EXCHANGE_RATES_STORAGE_KEY));
```
Comment (Overtorment) [inline blue_modules/currency.js:100]: looks like the bug caught. 
i wonder why did it work before...

**Comment by Overtorment** [comment]
wow, looks like a lot of work went into this PR  👏👏👏
